src/client/content_cao.cpp

```diff
@@ -725,6 +725,14 @@ void GenericCAO::addToScene(ITextureSource *tsrc)
 				addAnimatedMeshSceneNode(mesh, m_matrixnode);
 			m_animated_meshnode->grab();
 			mesh->drop(); // The scene node took hold of it
+
+			if (!checkMeshNormals(mesh)) {
+				infostream << "GenericCAO: recalculating normals for mesh "
+					   << m_prop.mesh << std::endl;
```

Comment:
Not sure if we use tabs+spaces for alignment, as the final indentation would vary depending on tab size. Continuation lines are usually indented by just one extra tab, but if you really want alignment, (if I may) I suggest using spaces right from where `infostream` starts.

Reply:
I just removed spaces. To be honest I think if auto-formatter and linter don't care, we shouldn't either.

Reply:
Thanks. The following is the only reason I suggested a change:

> tabs+spaces [...] as the final indentation would vary depending on tab size

A tab size of 8, for example, would break the alignment, which is why I suggested against this. A linter is after all just a program, humans have better judgement. :)

Reply:
Indentation is strictly tabs only in MT codestyle.

Reply:
Also beware that the Clang Format build check very often makes insane formatting requests that would make code less readable, you should overrule it with your good human judgement.